_data/sites.json

```diff
@@ -10423,6 +10423,16 @@
         ]
     },
 
+    {
+        "name": "Windscribe",
+        "url": "https://windscribe.com/myaccount",
+        "difficulty": "easy",
+        "notes": "Go to your account page, the delete button is small, you can find it under the 'joining date'. Also, their privacy policy states they'll store your username, password, email(if provided) even after account deletion.",
```

Comment:
I don't know what the service is about, but which other user data would it contain aside from the ones you mentioned?

If it keeps too much, we typically raise the difficulty

Reply:
Windscribe is a VPN service, technically it doesn't even require a email address, but you can provide one for password reset. Otherwise it's a mostly logless service and doesn't store any other data as they've mentioned in their privacy policy.
https://windscribe.com/privacy

Reply:
Ah, I'm ok with it then, thanks for providing the details!